test-requirements.txt

```diff
@@ -8,4 +8,5 @@ pytest-cov
 pytest-xdist
 tox
 twine
-wheel
\ No newline at end of file
+wheel
+pyyaml
```

Comment:
Missing newline at end of file.

I know it was already broken, but let's fix it

Reply:
True, I have fixed it